stable/spinnaker/values.yaml

```diff
@@ -31,14 +31,14 @@ mail:
 
 # Images for each component
 images:
```

Comment:
It's already a different commit, what does it matter? Also is there a policy against this? If not....